Resources/Prototypes/Catalog/Fills/Crates/armory.yml

```diff
@@ -48,3 +48,14 @@
       amount: 2
     - id: MagazinePistol
       amount: 4
+
+- type: entity
+  id: CrateRevolver
```

Comment:
id should have rex since there are other revolvers in the game

Reply:
> id should have rex since there are other revolvers in the game

I can do CrateRevolverRex, it's all right?

Reply:
ye